common/src/main/java/tds/common/configuration/EventLoggerConfiguration.java

```diff
@@ -31,6 +31,6 @@ public EventLoggerConfiguration(final ApplicationContext applicationContext,
   @Bean
   public MappedInterceptor eventInterceptor()
   {
-    return new MappedInterceptor(null, new EventLoggerInterceptor(applicationContext.getId(), objectMapper));
+    return new MappedInterceptor(new String[] {"/**"}, new String[] {"/health*"}, new EventLoggerInterceptor(applicationContext.getId(), objectMapper));
```

Comment:
I read it the same way at first but I'm assuming the first is an inclusion and the second parameter an exclusion.